team-telnyx/webrtc-examples
# feat(ENGDESK-19054): add new angular project and create a simple web dialer

- add new angular project and create a simple web dialer

This project was based on the official angular bootstrap sample

https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.ts
https://angular.io/start


Steps to test:

1 - copy `environment.sample.ts` and create a new one `environment.ts`
2 - Filled the object values inside `environment.ts`
3 - Run `yarn && yarn start`
4 - Click on `Connect` and Click on `Call`
5 - You should hear the call on the browser

![image](https://user-images.githubusercontent.com/16343871/192605511-37e8189a-2cc8-4cb9-9240-9bfb14fe792c.png)
